cmd/cloud_login.go

```diff
@@ -79,18 +77,6 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
-	// We want to use this fully consolidated config for things like
-	// host addresses, so users can overwrite them with env vars.
-	consolidatedCurrentConfig, warn, err := cloudapi.GetConsolidatedConfig(
```

Comment:
The consolidated config was used to initialize the `cloudapi.Client`, no longer needed.